content/javascript/concepts/arrays/terms/copyWithin/copyWithin.md

```diff
@@ -0,0 +1,87 @@
+---
+Title: '.copyWithin()'
+Description: 'Copies part of an array to another location in the same array and returns it without modifying its length.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Arrays'
+  - 'Methods'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `.copyWithin()` copies part of an array to another location in the same array and returns it without modifying its length.
+
+## Syntax
+
+The `.copyWithin()` method is a mutating method. It does not alter the length of the array, but it will change the content of the array and create new properties or delete existing properties, if necessary.
```

Comment:
This sentence seems to belong in the introduction rather than in the syntax section. I'd move all of line 19 to after line 15.